src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -26,19 +26,20 @@ export const PartyMap: React.FC = () => {
   >();
 
   const venue = useSelector(partyMapVenueSelector);
-  const usersInCamp = usePartygoers();
+  const { recentWorldUsers } = useRecentWorldUsers();
 
+  // @debt Separate counting logic into the hook
   const attendances = useMemo(
     () =>
-      usersInCamp
-        ? usersInCamp.reduce<Record<string, number>>((acc, value) => {
-            Object.keys(value.lastSeenIn).forEach((key) => {
-              acc[key] = (acc[key] || 0) + 1;
+      recentWorldUsers
+        ? recentWorldUsers.reduce<Record<string, number>>((acc, value) => {
```

Comment:
This is going to change the logic, as previously, `usePartygoers` would only include people in this venue, whereas `worldUsers` expands that concept.